Makefile

```diff
@@ -1,5 +1,9 @@
 -include config.mk
 
+help: ## Print help for targets with comments.
+	@printf "For more targets and info see the comments in the Makefile.\n\n"
+	@grep -E '^[a-zA-Z0-9._-]+:.*?## .*$$' Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
```

Comment:
Can you add a line break to keep it < 81 columns?

Reply:
Sure.